maintainers/maintainer-list.nix

```diff
@@ -13414,4 +13414,22 @@
     github = "jpagex";
     githubId = 635768;
   };
+  unrooted = {
```

Comment:
Yeah this list is not sorted currently, but I think most of the entries are alphabetically sorted, would you like to move your entries to the right place following the alphabetical order?